### PR TITLE
Licensing, Versioning and Meta data standardization and correction

### PR DESCRIPTION
**Pull request by anivar**
Changelog.
- Corrected and Standardized versioning and metadata 
- Added unifomity to Copyright and other fields. 
- Corrected wrong license information by analysing historical data and license in the font as base
- Moved Bulk text to Readme file.

### CODE REVIEW

File: Kalyani/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+
+History
+=====
+
+This font is the first Unicode font in Malayalam. 
+Initially it was named as Malayalam . 
+It was created by Shaji N V in 2002 from TeX archive font created by Jeroen Hellingman using a tool named Pfaedit.   In 2003 Baiju M added GSUB tables to the font and renamed it as MalOtf.ttf 
+This was the default font in Sopananam  GNU/Linux, (first GNU/Linux Custom destro with Malayalam Intefaces) released as a part of  UNDP-KBIP-FSFIndia project 
+In 2008 Hiran Venugopalan revamed font and added missing glyphs and featuretables.
```
Comment (santhoshtr) [inline Kalyani/README.md]: typo- renamed

featuretables -> feature tables

File: Kalyani/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+
+History
+=====
+
+This font is the first Unicode font in Malayalam. 
+Initially it was named as Malayalam .
```
Comment (santhoshtr) [inline Kalyani/README.md]: Remove space before .

File: Kalyani/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+
+History
+=====
+
+This font is the first Unicode font in Malayalam. 
+Initially it was named as Malayalam . 
+It was created by Shaji N V in 2002 from TeX archive font created by Jeroen Hellingman using a tool named Pfaedit.   In 2003 Baiju M added GSUB tables to the font and renamed it as MalOtf.ttf 
+This was the default font in Sopananam  GNU/Linux, (first GNU/Linux Custom destro with Malayalam Intefaces) released as a part of  UNDP-KBIP-FSFIndia project
```
Comment (santhoshtr) [inline Kalyani/README.md]: Interface - typo

File: Kalyani/README.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+History
+=====
+
+This font is the first Unicode font in Malayalam. 
+Initially it was named as Malayalam . 
+It was created by Shaji N V in 2002 from TeX archive font created by Jeroen Hellingman using a tool named Pfaedit.   In 2003 Baiju M added GSUB tables to the font and renamed it as MalOtf.ttf 
+This was the default font in Sopananam  GNU/Linux, (first GNU/Linux Custom destro with Malayalam Intefaces) released as a part of  UNDP-KBIP-FSFIndia project 
+In 2008 Hiran Venugopalan revamed font and added missing glyphs and featuretables. 
+
+Relevant Links
```
Comment (santhoshtr) [inline Kalyani/README.md:9]: relevant

File: Rachana/Rachana.sfd [hunk 5]
```diff
@@ -3,8 +3,8 @@ FontName: Rachana
 FullName: Rachana
 FamilyName: Rachana
 Weight: Book
-Copyright: Copyright (c) Rachana Akshara Vedi (Chitrajakumar R, Hussain KH, Rajeev Sebastian, Gangadharan N, Vijayakumaran Nair, Subash Kuraiakose ), 2005. \nCopyright 2013 Rajeesh K Nambiar, Santhosh Thottingal, Swathanthra Malayalam Computing (http://smc.org.in, http://savannah.nongnu.org/projects/smc/)
-Version: Rachana:06;Revision:0.0
+Copyright: Digitized data copyright(c) 2005 Rachana Akshara Vedi (Chitrajakumar R, Hussain KH, Rajeev Sebastian, Gangadharan N, Vijayakumaran Nair, Subash Kuraiakose), 2006-2014 Suresh P, Santhosh Thottingal, Rajeesh K Nambiar, Swathanthra Malayalam Computing (http://smc.org.in)
+Version: 6.11
```
Comment (santhoshtr) [inline Rachana/Rachana.sfd:7]: 6.1.1?

File: RaghuMalayalamSans/README [hunk 6]
```diff
@@ -0,0 +1,7 @@
...
+
+This first Version called RaghuMalayalam font Original Version 1.00 (2005) is a non-hinted text face, to be used for continuous text in Malayalm language. And is recommended to be used in minimum 14pt. size on screen and 10pt. size on paper (see the README file). The designed letterforms of RaghuMalayalam font in Malayalam script, have elegant structures of thick and thin strokes. Vertical strokes contain a slight curvature and scooped ends.
+
+RaghuMalayalam font Original Version 1.00 (2005) has been designed and developed, by Prof. R. K. Joshi (TypeFont Design Director, Visiting Design Specialist at C-DAC Mumbai) in association with Mr. Rajith Kumar K. M. (TypeFont Designer), assisted by Mr. Nirmal Biswas, Ms. Jui Mhatre and Ms. Supriya Kharkar at C-DAC Mumbai (formerly NCST).
+
+In 2008,  developers from swathanthra Malayalam Computing started maintaining the font as a tribute to him. Suresh P  modified opentype tables and added more glyphs to make it compatible with modern rendering systems .
```
Comment (santhoshtr) [inline RaghuMalayalamSans/README]: no space needed before fullstop

File: Kalyani/Kalyani.sfd [hunk 7]
```diff
@@ -3,8 +3,8 @@ FontName: Kalyani
 FullName: Kalyani
 FamilyName: Kalyani
 Weight: Book
-Copyright: Created by Shaji N Vyapron\n<nvshaji@netscape.net> with PfaEdit from Orginal TeX fonts for Malayalam (C) Jeroen Hellingman <jehe@kabelfoon.nl>\nLicensed under GNU GPL as MalOtf.\n\nConverted to SMC Meera04 font format by Hiran Venugopalan <hiran.v@gmail.com> 
-Version: Version 1.3
+Copyright: Digitized data copyright(c) 2002, Shaji N Vyapron\n<nvshaji@netscape.net>(featuring), Jeroen Hellingman <jehe@kabelfoon.nl> ,2003 Baiju M . 2008 Hiran Venugopalan <hiran.v@gmail.com>, Swathanthra Malayalam Computing (http://smc.org.in)
```
Comment (santhoshtr) [inline Kalyani/Kalyani.sfd]: Remove space before . and ,

**Comment by anivar** [comment]
Corrected typos

**Pull request merged**